File: python/meteroid/models/payment_method_info.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel
from .payment_method_type_enum import PaymentMethodTypeEnum


@dataclasses.dataclass
class PaymentMethodInfo(BaseModel):
    payment_method_type: PaymentMethodTypeEnum

    account_number_hint: t.Optional[str] = None

    card_brand: t.Optional[str] = None

    card_last4: t.Optional[str] = None

File: go/payment_method_info.go
// this file is @generated
package meteroid

type PaymentMethodInfo struct {
	AccountNumberHint *string `json:"account_number_hint,omitempty"`

	CardBrand *string `json:"card_brand,omitempty"`

	CardLast4 *string `json:"card_last4,omitempty"`

	PaymentMethodType PaymentMethodTypeEnum `json:"payment_method_type"`
}
